### ISSUE TITLE
Live Streaming

### ISSUE THREAD

**Issue by SamuelNitsche** [sampotts/plyr#1314]
### Expected behaviour

Progress bar and current time are hidden (Like described in #1072)

### Actual behaviour

Progress bar and current time are visible (Also like described in #1072)

### Steps to reproduce

Set live stream as source for player

### Environment

- Browser: Google Chrome
- Version: 71.0.3578.98 (Latest)
- Operating System: macOS
- Version: 10.14 (mojave)

### Console errors (if any)

None

### Link to where the bug is happening

https://codepen.io/anon/pen/GPzbWQ?editors=1010

**Comment by sampotts** [comment]
Why not hide them via the options?

**Comment by SamuelNitsche** [comment]
Yeah, of course I could (and actually do). But I wanted to report this because #1075 *should* have solved this "issue".

**Comment by mahogben** [comment]
Hiding with options would be preferred. Might be jumping in a bit late, but the stream you were using contains an #EXT-X-ENDLIST. Which provides the stream with a duration, that makes the changes in #1075 not apply and displays the controls as expected.

**Comment by JustinWebDev** [comment]
The progress bar is _almost_ useless for a live stream, but it does visibly indicate status of the stream (loading, playing).  I like the current time because it tells me how long I've been streaming the media.
